internal/server/server.go

```diff
@@ -264,6 +264,9 @@ func (s *Server) Start(ctx context.Context, param Param) error {
 		return err
 	}
 
+	if runtimeAPI := os.Getenv("AWS_LAMBDA_RUNTIME_API"); runtimeAPI != "" {
```

Comment:
Let's not assume that the user always wants to run Cerbos as an extension and let them opt into it via a command-line flag/environment variable.